README.md

```diff
@@ -8,26 +8,26 @@ The courtroom suite includes an abstract generic [SwearGame contract](#sweargame
 
 ### SwearGame contract
 
-This is the main general swear contract which conduct the trial.
+This is the main general swear contract which conducts the trial.
 
 It calls the specific [trial rules contract](#trial-rules-contract) to get the [witnesses](#witness-contract),get the trial statuses and transitions
 and to check for expiry time for a specific case and trial status.
```

Comment:
This whole paragraph needs revisiting. It is not clear and **uses references to terms a reader is not aware of**.